c-dependencies/js-compute-runtime/builtins/crypto-key.h

```diff
@@ -0,0 +1,942 @@
+#pragma once
```

Comment:
Could you switch to header guards? If we ever change build systems, `#pragma once` could become problematic.